src/test/java/uk/gov/register/functional/DataDownloadFunctionalTest.java

```diff
@@ -29,7 +29,7 @@
 public class DataDownloadFunctionalTest {
 
     @ClassRule
-    public static RegisterRule register = new RegisterRule("address");
+    public static RegisterRule register = new RegisterRule();
```

Comment:
Kapow same as above - a possible change to`@Rule` without the static for consistency?